recipe/meta.yaml

```diff
@@ -24,20 +24,20 @@ requirements:
   run:
     - python >=3.5
     - pyzmq >=17.1.2
-    - typeguard
+    - typeguard >=2.5
     - ipyparallel
     - globus-sdk
     - dill
     - tblib
     - ipykernel
     - requests
     - paramiko
-    - psutil
-    - sqlalchemy ==1.3.3
+    - psutil >=5.5.1
+    - sqlalchemy >=1.3.0,!=1.3.4
```

Comment:
@conda-forge/parsl, this is the main change, so make sure you agree that this is correct.  The other package constraints should be the same as upstream as well, but feel free to check.

Reply:
This looks good.